packages/db-ra-data-rest/vite.config.js

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import path from "node:path";
+
+export default defineConfig({
+  // https://github.com/vitejs/vite/issues/6215#issuecomment-1076980852
+  plugins: [react({ jsxRuntime: "classic" })],
```

Comment:
Do we need the `react` plugin at all?

Reply:
it seemed so from what I've seen online and I remember receiving some error without it! I'll post here which one was it

Reply:
No, no error at all!
I honestly have been confused thinking that this lib had some peerdeps over react... but it doesn't, so all good 😃

Reply:
less is more!!! :)

Reply:
It's the mastery of simplicity which is the greatest skill to have